Tests/AdyenTests/Adyen Tests/Components/Redirect/RedirectComponentTests.swift

```diff
@@ -192,33 +192,6 @@ class RedirectComponentTests: XCTestCase {
         waitForExpectations(timeout: 10, handler: nil)
     }
 
-    func testOpenHttpWebLinkAndClose() {
```

Comment:
This is a useful test 🤔

Reply:
its not the point, it was failing and can't figure out why, we can try to fix it in another PR